CHANGES.rst

```diff
@@ -49,6 +49,8 @@ Imviz
 Mosviz
 ^^^^^^
 
+- R-grism data are now loaded with the correct orientation. [#1619]
```

Comment:
I recommend including a "2D" descriptor to be more specific about the resolved issue.

Reply:
good idea, added to the changelog and will merge once CI passes.  Thanks!